packages/react-magma-dom/src/components/StyledButton/styles.ts

```diff
@@ -297,28 +367,45 @@ export function buildActiveColor(props) {
   props = buildPropsWithDefaultButtonStyles(props);
 
   if (props.color === 'marketing') {
+    if (props.variant !== 'solid' && props.isInverse) {
+      return props.theme.colors.tertiary500;
```

Comment:
Can this marketing check be moved to the switch statement on line 394?

Reply:
Done